packages/theme-data/src/colorSchemes/webLight/unresolvedRoles.js

```diff
@@ -38,6 +38,9 @@ const webLightThemeResolvedRoles = extendTheme(lightGrayTheme.unresolvedRoles, {
     value: {
       ref: "colorScheme.accentColor"
     }
+  },
+  "flyout.borderColor": {
+    value: "#d4dbe1"
```

Comment:
Mind defining this value in the `oldBasics` object above to keep things tidy?

Reply:
got it, didn't realize there was that object.